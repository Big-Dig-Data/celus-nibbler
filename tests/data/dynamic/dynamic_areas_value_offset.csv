MY NAME,

Title1,1
Title2,2
Title3,3

Sessions
2020-01,1
2020-02,2
2020-03,3
