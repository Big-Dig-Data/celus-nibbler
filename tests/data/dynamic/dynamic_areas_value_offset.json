{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "dynamic_areas_value_offset",
  "data_format": {
    "name": "simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "possible_row_offsets": [0, 1],
  "heuristics": {
    "conds": [
      {
        "pattern": "MY NAME",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
          {
            "role": "value",
            "source": {
               "coord": {
                 "row": 1,
                 "col": 1
               },
               "direction": "down"
            }
          }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 1
          },
          "direction": "right"
        },
        "condition": {
          "conds": [
            {
              "pattern": "Sessions",
              "coord": {
                "row": 0,
                "col": 0
              },
              "kind": "regex"
            }
          ],
          "kind": "and"
        }
      },
      "metrics": {
        "source": {
          "row": 0,
          "col": 0
        }
      },
      "dates": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        }
      },
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.generic"
    }
  ]
}
