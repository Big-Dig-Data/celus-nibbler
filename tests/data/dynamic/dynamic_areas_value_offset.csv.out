"2020-01-01","2020-01-31","","","","","","","","","Sessions","1"
"2020-02-01","2020-02-29","","","","","","","","","Sessions","2"
"2020-03-01","2020-03-31","","","","","","","","","Sessions","3"
